src/provider.c

```diff
@@ -1532,7 +1535,7 @@ int OSSL_provider_init(const OSSL_CORE_HANDLE *handle, const OSSL_DISPATCH *in,
             } else if (strncmp(str, "no-allowed-mechanisms", toklen) == 0) {
                 ctx->no_allowed_mechanisms = true;
             } else if (strncmp(str, "no-operation-state", toklen) == 0) {
```

Comment:
with this change, this option now affects also the `C_SetOperationState`. Should we update the documentation regarding this change?

Reply:
Set and Get are a pair the no_opertion_state flag always meant to block both (which are always used together in the code, currently) so I do not think we need a doc change for that.